cmd/connect.go

```diff
@@ -76,6 +99,29 @@ var connectCommand = Command{
 			return err
 		}
 
-		return cli.Connect(ctx, deviceID, targets)
+		remoteAccessTargets := make([]client.RemoteAccessConnection, 0)
+
+		if opts[connectConfigFileOption] != "" {
+			configBytes, err := os.ReadFile(opts[connectConfigFileOption])
+			if err != nil {
+				return fmt.Errorf("error reading config file: %w", err)
+			}
+
+			if err := json.Unmarshal(configBytes, &remoteAccessTargets); err != nil {
+				return fmt.Errorf("error parsing config file: %w", err)
+			}
+
+			if len(remoteAccessTargets) == 0 {
+				return fmt.Errorf("no connections defined in config file")
+			}
```

Comment:
If we add
```go
cli.ConnectMulti(ctx, remoteAccessTargets)
return nil
```

in the `if` block, then we can keep the rest of the file as-is and use the `cli.Connect()`

That would retain the command's non-zero exit code on errors when connecting to a single device.
Currently it always returns 0, even if nothing works.